<launch>
   <node name="lx_camera_node" pkg="lx_camera_ros" type="lx_camera_node" output="screen">
	<param name="ip" value="0" />
	<param name="is_show" value="0" />
	<param name="is_xyz" value="1" />
	<param name="is_depth" value="0" />
	<param name="is_amp" value="0" />
	<param name="is_rgb" value="0" />
	<param name="application" value="0" />
   </node>

</launch>
